modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase

```diff
@@ -26,29 +26,26 @@ definition {
 		}
 
 		task ("Employee user creates an organization") {
```

Comment:
Change to "Create an organization"

Reply:
I already update this on this [PR](https://github.com/is-solutions-delivery/liferay-portal/pull/4985)